pkg/api/addons.go

```diff
@@ -385,12 +391,10 @@ func assignDefaultAddonVals(addon, defaults KubernetesAddon, isUpdate bool) Kube
 	return addon
 }
 
-func synthesizeAddonsConfig(addons []KubernetesAddon, addon KubernetesAddon, enableIfNil bool, isUpdate bool) {
```

Comment:
I supposed `enableIfNil` was once used but is leftover cruft at this point (being that is only called above w/ static `false`.